### ISSUE TITLE
Error base_template_class is deprecated

### ISSUE THREAD

**Issue by sebastian-lenz** [craftcms/cms#4755]
### Description

When running Craft CMS the following php error is triggered:
```
PHP Deprecated: The "base_template_class" option on Twig\Environment is 
deprecated since Twig 2.7.0. 
in vendor/twig/twig/src/Environment.php on line 121
```

The option `base_template_class` is set in the View class [here](https://github.com/craftcms/cms/blob/develop/src/web/View.php#L1581). This is related to #4704.

This is a documented behavior of twig, please refer to their docs:
https://twig.symfony.com/doc/2.x/deprecated.html#environment

### Steps to reproduce

1. Open any page in Craft, the admin login page for example.
2. Check the error log of the web server or the console output of XDebug.

### Additional info

- Craft version: Craft Solo 3.2.10
- PHP version: 7.3.8
- Database driver & version: MySQL 5.7.27

**Comment by brandonkelly** [comment]
Great.

**Comment by sebastian-lenz** [comment]
Sorry for that one, I'm just the messenger.

**Comment by brandonkelly** [comment]
We primarily extend the base class to hook template rendering into Yii profiling for the Debug toolbar. There’s no good alternative way to do that currently, as far as I can tell.

Posted twigphp/Twig#3129 asking for a new way to hook into template rendering, so we’ll see what they come back with.

**Comment by brandonkelly** [comment]
This has been fixed for Craft 3.3!

**Comment by nikolowry** [comment]
@brandonkelly we use `suppressTemplateErrors` in production environments -- what's the recommended approach now that `suppressTemplateErrors` is deprecated?
